### DISCUSSION TITLE
Welcome to the TinyGo discussion!

### DISCUSSION THREAD

**Discussion by deadprogram** [tinygo-org/tinygo#3953] [Announcements]
We are going to try a little experiment with using the discussions feature a forum oriented way to interact with the TinyGo community.

I look forward to our shared interactions here!

**Comment by dkegel-fastly** [comment]
For those just tuning in, most discussion until now has been going on in #tinygo and #tinygo-dev, so if you don't find what you're looking for here, go check out those channels at http://gophers.slack.com; see also https://go.dev/help (especially if you need an invite to the gophers slack).

**Comment by Gustavomurta** [comment]
What is GitHub Discussions? A complete guide
_https://resources.github.com/devops/process/planning/discussions/_

GitHub Discussions brings community forums right to your public or private repositories on GitHub making it easy to communicate with your community, ask and answer questions, share updates, and more. Here’s what you need to know about GitHub Discussions, best practices, and how other open source projects are using it.

**Comment by Gustavomurta** [comment]
The link to discussions on Slack is in Guide - Contributing. 
_https://tinygo.org/docs/guides/contributing/_

What do you think about adding the link to discussions on Github itself?
_https://github.com/orgs/tinygo-org/discussions_
